Detectors/ITSMFT/MFT/base/src/Geometry.cxx

```diff
@@ -42,7 +42,8 @@ const Double_t Geometry::sSensorInterspace = 0.01; //[cm]  Offset between two ad
 const Double_t Geometry::sSensorSideOffset = 0.04; // [cm] Side Offset between the ladder edge and the chip edge
 const Double_t Geometry::sSensorTopOffset = 0.04; // [cm] Top Offset between the ladder edge and the chip edge
 const Double_t Geometry::sLadderOffsetToEnd = 4.7; // [cm] Offset between the last Chip and the end of the ladder toward the DAQ connector
-const Double_t Geometry::sSensorThickness = 50.e-4; // 50 microns
+const Double_t Geometry::sSensorThickness = 30.e-4; // 50 microns
+const Double_t Geometry::sChipThickness = 50.e-4; // 50 microns
 
 const Double_t Geometry::sHeightActive = 1.3;
 const Double_t Geometry::sHeightReadout = 0.2;
```

Comment:
These sHeightActive and sHeightReadout are not needed anymore, as far as I can see